### DISCUSSION TITLE
How do I add the stats to a gist? 💡

### DISCUSSION THREAD

**Discussion by ericviana** [athul/waka-readme#96] [Q&A] [answered]
Could anybody gimme a hint on how to add this to a gist?
Like this one: https://github.com/matchai/awesome-pinned-gists

**Comment by athul** [comment, marked as answer]
Hi,

Better use https://github.com/matchai/waka-box for a making a Gist with the wakatime data

**Reply by ericviana** [reply]
I'll be doing it, appreciate it, Athul
